Assets/Scripts/Map/MapPatrollingGen/WatchmanRouteSolver.cs

```diff
@@ -11,16 +11,16 @@ namespace Maes.Map.MapPatrollingGen
 {
     public static class WatchmanRouteSolver
     {
-        public static PatrollingMap MakePatrollingMap(SimulationMap<Tile> simulationMap)
+        public static PatrollingMap MakePatrollingMap(SimulationMap<Tile> simulationMap, bool colorIslands)
```

Comment:
```suggestion
        public static PatrollingMap MakePatrollingMap(SimulationMap<Tile> simulationMap, bool colorIslands = false)
```
Set the default to false, only if we are debugging something about the map gen do we actually want to see it